res/ui/Tactile/Components/instruction_text.qml

```diff
@@ -15,7 +15,7 @@ import QtQuick 2.0
 Text {
```

Comment:
You need an entry in [`debian/changelog`](https://github.com/KanoComputing/tactile/blob/40526b47dee2bd771655d5d8dfb845fff2feca30/debian/changelog#L1-L11) for this change so that the Debian package has the correct version. It should look something like:

```
tactile (4.3.0-0) unstable; urgency=low

  * Change default text formatting to normal

 -- Team Kano <dev@kano.me>  Tue, 12 Mar 2019 15:35:00 +0000

tactile (4.2.0-0) unstable; urgency=low

  * Fixed minor spelling errors
 ...
```